tests.py

```diff
@@ -44,7 +85,7 @@ def target(self):
9999999999998pt" height="12.524999999999999pt" viewBox="0 0 27.009999999999998 12.524999999999999"><metadata><pmml2svg:baseline-shift>3.6949999999999985</pmml2svg:baseline-shift></metadata><g stroke="none" fill="#000000" text-rendering="optimizeLegibility" font-family="STIXGeneral,STIXSizeOneSym,STIXIntegralsD,STIXIntegralsSm,STIXIntegralsUp,STIXIntegralsUpD,STIXIntegralsUpSm,STIXNonUnicode,STIXSizeFiveSym,STIXSizeFourSym,STIXSizeThreeSym,STIXSizeTwoSym,STIXVariants"><g xmlns:doc="http://nwalsh.com/xsl/documentation/1.0" style="font-family: STIXGeneral,STIXSizeOneSym,STIXIntegralsD,STIXIntegralsSm,STIXIntegralsUp,STIXIntegralsUpD,STIXIntegralsUpSm,STIXNonUnicode,STIXSizeFiveSym,STIXSizeFourSym,STIXSizeThreeSym,STIXSizeTwoSym,STIXVariants; fill: ; background-color: transparent; "><g style="font-family: STIXGeneral,STIXSizeOneSym,STIXIntegralsD,STIXIntegralsSm,STIXIntegralsUp,STIXIntegralsUpD,STIXIntegralsUpSm,STIXNonUnicode,STIXSizeFiveSym,STIXSizeFourSym,STIXSizeThreeSym,STIXSizeTwoSym,STIXVariants; fill: ; background-color: transparent; "><g style="font-family: STIXGeneral,STIXSizeOneSym,STIXIntegralsD,STIXIntegralsSm,STIXIntegralsUp,STIXIntegralsUpD,STIXIntegralsUpSm,STIXNonUnicode,STIXSizeFiveSym,STIXSizeFourSym,STIXSizeThreeSym,STIXSizeTwoSym,STIXVariants; fill: ; background-color: transparent; "><text style="font-family: STIXGeneral, STIXSizeOneSym, STIXIntegralsD, STIXIntegralsSm, STIXIntegralsUp, STIXIntegralsUpD, STIXIntegralsUpSm, STIXNonUnicode, STIXSizeFiveSym, STIXSizeFourSym, STIXSizeThreeSym, STIXSizeTwoSym, STIXVariants; fill: black; background-color: transparent; " x="2" y="8.83" font-size="10">sin</text><g style="font-family: STIXGeneral,STIXSizeOneSym,STIXIntegralsD,STIXIntegralsSm,STIXIntegralsUp,STIXIntegralsUpD,STIXIntegralsUpSm,STIXNonUnicode,STIXSizeFiveSym,STIXSizeFourSym,STIXSizeThreeSym,STIXSizeTwoSym,STIXVariants; fill: ; background-color: transparent; "><g style="font-family: STIXGeneral, STIXSizeOneSym, STIXIntegralsD, STIXIntegralsSm, STIXIntegralsUp, STIXIntegralsUpD, STIXIntegralsUpSm, STIXNonUnicode, STIXSizeFiveSym, STIXSizeFourSym, STIXSizeThreeSym, STIXSizeTwoSym, STIXVariants; fill: black; background-color: transparent; "><text x="13.65" y="8.764999999999999" font-size="10">(</text></g><text style="font-family: STIXGeneral, STIXSizeOneSym, STIXIntegralsD, STIXIntegralsSm, STIXIntegralsUp, STIXIntegralsUpD, STIXIntegralsUpSm, STIXNonUnicode, STIXSizeFiveSym, STIXSizeFourSym, STIXSizeThreeSym, STIXSizeTwoSym, STIXVariants; font-style: italic; fill: black; background-color: transparent; " x="17.23" y="8.83" font-size="10">x</text><g style="font-family: STIXGeneral, STIXSizeOneSym, STIXIntegralsD, STIXIntegralsSm, STIXIntegralsUp, STIXIntegralsUpD, STIXIntegralsUpSm, STIXNonUnicode, STIXSizeFiveSym, STIXSizeFourSym, STIXSizeThreeSym, STIXSizeTwoSym, STIXVariants; fill: black; background-color: transparent; "><text x="21.689999999999998" y="8.764999999999999" font-size="10">)</text></g></g></g></g></g></g></svg>"""
```

Comment:
The values ``xmlns:pmml2svg="https://sourceforge.net/projects/pmml2svg/" xmlns="http://www.w3.org/2000/svg"`` were switched in my latest code update for some reason. Will this effect anything?